npm/private/npm_package.bzl

```diff
@@ -506,14 +506,6 @@ def stamped_package_json(name, stamp_var, **kwargs):
 
     For more information on stamping, read https://docs.aspect.build/rules/aspect_bazel_lib/docs/stamping.
 
-    Using this rule requires that you register the jq toolchain in your WORKSPACE:
```

Comment:
Isn't this still true? However there's now just another way of doing it?

Reply:
It will get registered implicitly in rules_js_configure now so there is no need to document all of the required toolchains in each rule docstring. jq was a one off here and we don't do this for other toolchains required for other rules.